readme.md

```diff
@@ -62,52 +62,39 @@ Pass `false` to not set `rel`s on links.
 > When using a `target`, add [`noopener` and `noreferrer`][mdn-a] to avoid
 > exploitation of the `window.opener` API.
 
+###### `options.protocols`
+
+Protocols used to filter against the [is-absolute](https://www.npmjs.com/package/is-absolute) package.
```

Comment:
I don’t think there’s a need to mention is-absolute. That’s internals. I suggest mentioning that people can allow other protocols here, such as `mailto`, or others, instead?

Reply:
Sure thing, pushed in my latest commit.